models/user.go

```diff
@@ -131,14 +131,21 @@ type User struct {
 	// Maximum repository creation limit, -1 means use global default
 	MaxRepoCreation int `xorm:"NOT NULL DEFAULT -1"`
 
-	// Permissions
-	IsActive                bool `xorm:"INDEX"` // Activate primary email
-	IsAdmin                 bool
-	IsRestricted            bool `xorm:"NOT NULL DEFAULT false"`
+	// IsActive true: primary email is activated, user can access Web UI and Git SSH.
+	// false: an inactive user can only log in Web UI for account operations (ex: activate the account by email), no other access.
+	IsActive bool `xorm:"INDEX"`
+	// user is a Gitea admin, can access all repositories and admin pages
+	IsAdmin bool
+	// true: user is allowed to see only organizations/repositories that he has explicit rights specified
```

Comment:
```suggestion
	// true: user is only allowed to see organizations/repositories that they has explicit rights to.
```

Reply:
Still a syntax error "they has", so I use "he/she has" now. 😊

Reply:
Oh sorry, I am not a native speaker, just realized that `singular they` is correct for gender-neutral language (I googled it from Wikipedia)